python/matlab_converter.py

```diff
@@ -0,0 +1,23 @@
+from analysis import *
+from utils import *
+import matplotlib.pyplot as plt 
+import numpy as np 
+import matplotlib.animation as animation
+import os
+import pandas as pd
+import scipy.io
+
+os.chdir("./")
```

Comment:
Is this necessary? Maybe mark a TODO here to remove this in the future once the project has a proper setup script.